src/js/components/Sidebar/stories/Tooltips.js

```diff
@@ -20,54 +21,44 @@ import {
   Stakeholder,
 } from 'grommet-icons';
 
-import { Sidebar } from '../Sidebar';
+import { deepMerge } from 'grommet/utils';
 
 const src = '//s.gravatar.com/avatar/b7fb138d53ba0f573212ccce38a7c43b?s=80';
 
-const NotificationIcon = () => (
-  <Stack anchor="top-right">
-    <Notification />
-    <Box background="accent-1" pad="xsmall" round responsive={false} />
-  </Stack>
-);
+const customTip = deepMerge(grommet, {
```

Comment:
I'm not sure this is a good type of guidance as it seems specific to the Sidebar scenario. Are you thinking all uses of Tip would be shown in this style?

Reply:
I've removed the `deepMerge`  and the `grommet` theme usage for better guidance.